### DISCUSSION TITLE
Auto generation by copilot isn't working for me.

### DISCUSSION THREAD

**Discussion by AswinSuku** [skills/.github#291] [Q&A] [answered]
**I havent't subscribed to copilot free trial plan ($10/month OR $100/year after trial period) so is it because of that auto-suggestion isn't working for me. I have also tried Ctrl+Enter but not working**

**Comment by racheltienng** [comment]
Are you in Step 2?

After putting the codes to commit and push in the TERMINAL next to the Debug Console on the lower side:
![Screenshot 2023-07-18 at 16 25 46](https://github.com/skills/.github/assets/65115880/a26a342c-800a-492f-96c5-18a6bd59d23c)


Then you can workaround by:
Go to your main GitHub-CoPilot page
Click folder Steps
Click on 3-Copilot-Hub


![Screenshot 2023-07-18 at 16 23 13](https://github.com/skills/.github/assets/65115880/820dd605-b65d-40f7-8b7a-04cf1a2ee4e4)

I agree that the instructions in the activity in Step 2 create confusion. I had to search around 30mins where the Terminal was. And after refreshing I couldn't go to Step 3. Had to work around and find it myself.

**Reply by AswinSuku** [reply]
I am in step 4 , it is not giving me auto-suggestions while coding. The same thing happened to me in step 2 & 3

**Reply by racheltienng** [reply, marked as answer]
Same thing happened to me, I rebooted and try Ctrl+Enter again and it worked. If you're on Mac try Ctrl (^) + Enter.

**Reply by AswinSuku** [reply]
thanks rachel

**Comment by The11sun** [comment]
ขอบคุณมากรู้สึกว่าผมจะสมัครผลิตภัณฑ์ของคุณไปเยอะเหลือเกินจนจำไม่ได้ฉันจำไม่ได้ว่าผมไปสมัครอะไรไปบ้างทุกวันนี้ผมไม่ยกเลิกการสมัครหรอกอย่างน้อยๆข้อความพวกคุณมันไม่ได้ทำร้ายผมถึงแม้สิ่งที่คุณพยายามส่งให้ผมได้เรียนรู้ได้ศึกษาผมจะไม่เข้าใจเลยผมจะไม่ได้รู้อะไรเลยแต่อย่างน้อยก็เป็นความสุขทางใจอย่างหนึ่งและผมรู้ว่าพวกคุณทำอะไรก็เก่งขนาดไหนผมอยากเป็นคนนึงที่สามารถที่จะทำเหมือนอย่างพวกคนที่ทำให้คุณทำได้ผมยกย่องที่นี่ผมยกย่ององค์กรที่ชื่อทับในความคิดผมคิดว่าที่นี่คือโรงเรียนสอนสอนพ่อมดตัวจริงผมเคยเห็นแต่ในนิยายในแล้วนิยายเก่าๆพ่อมดแต่ผมรู้ว่ามันมีจริงในโลกใบนี้ก็คือที่นี่แต่ถ้าให้ผมเข้าไปเป็นส่วนของความเป็นทำงานอย่างผมทำไม่ได้หรอกในตอนนี้มันเป็นความหวังเป็นความฝันที่มันไกลตัวในตอนนั้นซึ่งจริงๆแล้วผมต้องเอาตัวให้รอดเอาตัวให้รอดก่อนก่อนที่จะฝันก่อนที่จะหวังไกลขนาดนั้นซึ่งในส่วนทุกย่างก้าวอยู่งานมันมีความยากลำบากมีแต่ความสูญเสียมีแต่สิ่งที่มันมีผลกระทบทั้งนั้นจนผมท้อแล้วท้ออีกท้อจนเคยให้กำลังใจตัวเองแต่มันเหมือนกันโกหกตัวเองเรื่อยมาเพราะผมชีวิตในแต่ละวันมันยังผ่านมันก้าวผ่านมันไปไม่ได้มันไม่สามารถที่จะมีบันไดขั้นต่อไปได้เอง

ในวันที่ พ. 19 ก.ค. 2023 18:36 น. AswinSuku ***@***.***>
เขียนว่า:
